### PR TITLE
setup repo with required root files

### PR DESCRIPTION
**Pull request by chrisantonellis**
following wayfair documentation on how to setup an open source repo:

* You must have a CODE_OF_CONDUCT.md that underlines the behavior expected of maintainers, contributors, and users. You can find a template for your Code of Conduct here
* You must create a meaningful README.md that includes prerequisites, running tests, and versioning
* You must include a CONTRIBUTING.md file
* All projects must include a MAINTAINERS.md listing and at least one active maintainer by name and an obfuscated contact email (e.g. nvlatko [at] wayfair.com)
* You must create a license file and state the specific, approved license that applies to all code owned by Wayfair. If you use third-party OSS, you must only use license-compatible projects/software. You must honor the original licenses used in those third-party projects in the license file. For more information and guidance, contact Natali Vlatko for support on licensing
* You should semantically version project artefacts. You must tag all versions in GitHub with the exact version name: e.g., 0.1.0.
* You should scrub your Git history if your project is moving out of GitHub Enterprise and into our public GitHub organization
* You must create a SECURITY.md file in the main root folder of your repository. This text is sufficient for the file: “If you have discovered a security vulnerability, please email opensource@wayfair.com”
* Your repositories must not, at any time, include Wayfair specifics such as credentials and private identifiers (e.g. no references to csnzoo)
* You should review all merge requests for implanted security backdoors and vulnerable code fragments regardless of who is making the pull request

### CODE REVIEW

**Review by jdw6359 — APPROVED** [review]
`CONTRIBUTING` is empty - you could probably use [columbo's contributing guide](https://github.com/wayfair-incubator/columbo/blob/main/CONTRIBUTING.md) as a basis for yours.

everything else lgtm

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright 2020  Wayfair, Inc.
```
Comment (jamescurtin) [inline LICENSE:1]:
```suggestion
Copyright 2021  Wayfair, Inc.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Changelog
```
Comment (jamescurtin) [inline CHANGELOG.md:1]: `CONTRIBUTING.md` is empty so leaving a comment here instead: 

Could you add content to the contributing guide? Columbo has a good example that can be re-used.

**Review by jamescurtin — COMMENTED** [review]
Excited to see this launch!

**Pull request merged**